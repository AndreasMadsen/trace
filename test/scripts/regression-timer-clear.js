'use strict';

const a = setTimeout(function () {
  throw new Error('setTimeout callback should not be called');
}, 1);

clearTimeout(a);

const b = setInterval(function () {
  throw new Error('setInterval callback should not be called');
}, 1);

clearInterval(b);

const c = setImmediate(function () {
  throw new Error('setImmediate callback should not be called');
});

clearImmediate(c);
