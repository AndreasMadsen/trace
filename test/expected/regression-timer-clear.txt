warning: you are using async-hook which is unstable.
